services/frontend-service/src/ui/ReleaseDialog.tsx

```diff
@@ -120,15 +125,21 @@ const QueueDiff = (props: { queued: number; current: number; releases: Release[]
                 <span>
                     &nbsp;
                     {' queued: '}
-                    <span className="ahead">{'+' + diff}</span>
+                    <span className="ahead" data-testid="queue-diff">
+                        {'+' + diff}
+                    </span>
                 </span>
             </Tooltip>
         );
     }
     return (
         <Tooltip title={prefix + diff + ' behind'}>
             <span>
-                <span className="ahead">{'+' + diff}</span>
+                &nbsp;
+                {' queued: '}
+                <span className="ahead" data-testid="queue-diff">
```

Comment:
Is the className correct here? or should it be `behind`?

Reply:
yes! I totally overlooked the colors.
Done